### PR TITLE
new aab package

### PR DESCRIPTION
**Pull request by cav71**
This adds a new aab package from https://pypi.org/project/aab.

@conda-forge/help-python, ready for review!

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] ~If static libraries are linked in, the license of the static library is packaged.~
- [ ] ~Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).~
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/aab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pip check
+    - aab --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/glutanimate/anki-addon-builder
+  summary: Anki Add-on Builder
+  dev_url: https://github.com/glutanimate/anki-addon-builder
+  license: AGPL-3.0
```
Comment (xhochy) [inline recipes/aab/meta.yaml:43]: Please check for only vs or-later

File: recipes/aab/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six
+    - jsonschema
+    - whichcraft
+    - pathlib  # [py<34]
```
Comment (xhochy) [inline recipes/aab/meta.yaml:28]: You can make this package noarch by limiting to python>=3.6

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/aab/meta.yaml (hunk 1 above)
Comment (cav71) [reply recipes/aab/meta.yaml:43]: I don't understand, what do you mean with "check for only vs or-later"?

Comment (xhochy) [reply recipes/aab/meta.yaml:43]: These are the two valid forms of the AGPL-3.0 license identifier, either `AGPL-3.0-only` or `AGPL-3.0-or-later`.

Comment (cav71) [reply recipes/aab/meta.yaml]: done as AGPL-3.0-or-later

File: recipes/aab/meta.yaml (hunk 2 above)
Comment (cav71) [reply recipes/aab/meta.yaml]: I think it does with the latest changes

File: recipes/aab/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aab-{{ version }}.tar.gz
+  sha256: 1be9cd6dceac28c570df268a2aa20c1935a59043c88c9f4ce88a6d208e007d09
+
+build:
+  skip: true  # [py>=400]
```
Comment (xhochy) [inline recipes/aab/meta.yaml:13]:
```suggestion
  noarch: python
```

**Review by xhochy — APPROVED** [review]

**Pull request merged**